CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 ### Features
 - Remove MoPub header bidding support
+- Add ability to set Children's Online Privacy Protection Act ("COPPA") flag
```

Comment:
Could you add an E2E test checking that if Criteo singleton is setup with the flag, then the next CDB calls are actually containing the flag accordingly.

Reply:
E2E test have been added.
Resolving this conversation.